docs/undergrad.yaml

```diff
@@ -370,7 +370,12 @@ Single Variable Complex Analysis:
     antiderivative: ''
     complex exponential: 'complex.exp'
     extension of trigonometric functions to the complex plane: 'complex.sin'
-    power series expansion of elementary functions: ''
+    power series expansion of elementary functions:
+      cos: 'complex.has_sum_cos'
+      sin: 'complex.has_sum_sin'
+      log: ''
+      arctan: ''
+      arcsin: ''
```

Comment:
https://github.com/leanprover-community/leanprover-community.github.io/pull/311 should make this format legal, such that it renders on the site as `power series expansion of elementary functions (cos, sin)`

Reply:
Ah, this also needs #18407